### PR TITLE
Add replica creation

### PR DESCRIPTION
**Pull request by SiarheiShuliakouski**

### CODE REVIEW

**Review by vold029 — CHANGES_REQUESTED** [review]
main.tf line 19:
`count = "${var.module_enabled && var.replication_enabled ? 0 : 0}"` 
is it correct or should be 
`count = "${var.module_enabled && var.replication_enabled ? 1 : 0}"`

**Comment by SiarheiShuliakouski** [comment]
Hi vold029,

main.tf line 19 should be:

count = "${var.module_enabled && var.replication_enabled ? 0 : 0}"

File: main.tf [hunk 1]
```diff
@@ -46,3 +46,27 @@ resource "aws_elasticache_cluster" "redis-with-subnet-group-and-security-group"
 #   tags = "${merge(local.default_tags, var.tags)}"
 # }
 
+resource "aws_elasticache_replication_group" "example" {
```
Comment (ToROxI) [inline main.tf]: why resource named "example"?

File: main.tf [hunk 2]
```diff
@@ -16,7 +16,7 @@ resource "aws_elasticache_subnet_group" "this" {
 resource "aws_elasticache_cluster" "redis-with-subnet-group-and-security-group" {
   #   count = "${ var.module_enabled && "${lower(var.cache_engine)}" == "redis" && "${length(var.subnet_ids)}" > 0 && "${length(var.security_group_ids)}" > 0 ? 1 : 0 }"
 
-  count                = "${ var.module_enabled ? 1 : 0 }"
+  count = "${var.module_enabled && var.replication_enabled ? 0 : 0}"
```
Comment (ToROxI) [inline main.tf]: should be 
```suggestion
  count = "${var.module_enabled && var.replication_enabled ? 0 : 1}"
```

**Review by vold029 — APPROVED** [review]
Approve

**Pull request merged**